pelican/tests/test_log.py

```diff
@@ -80,3 +83,61 @@ def do_logging():
             self.assertEqual(
                 self.handler.count_logs('Another log \\d', logging.WARNING),
                 0)
+
+    def test_fatal_logger_no_quit(self):
+        # log.FatalLogger.fatal_log_level = logging.NOTSET
+        log.init(logging.DEBUG, fatal='')
+
+        for level in ('debug', 'info', 'warning', 'error', 'critical'):
+            self._log_action(level)('%s: FatalLogger.%s()' % (__name__, level))
+
+    def test_fatal_logger_warning(self):
+        # log.FatalLogger.fatal_log_level = logging.WARNING
+        log.init(logging.DEBUG, fatal='warning')
```

Comment:
This changes global state, particularly `FatalLogger.fatal_log_level`. You need to clear this after the test, otherwise tests running after this will work based on this value and can fail (as evident from the CI run). Easiest way would be to add a line in the `teadDown` method that puts the value back to its default:

```python
log.FatalLogger.fatal_log_level = logging.NOTSET
```

Reply:
That's kind of funny actually ;) Thanks, I'll get that under control.